envision/web/src/components/simulation.js

```diff
@@ -185,10 +185,11 @@ export default function Simulation({
           // React to trigger update, 0.1 millisecond
           await sleep(0.1);
         } else if (playingMode == PLAYMODES.near_real_time) {
-          await sleep(
-            msInSec * (elapsed_times[0] - prevElapsedTime) -
-              (Date.now() - waitStartTime)
-          );
+          let delt_elapsed_time_in_sec = msInSec * (elapsed_times[0] - prevElapsedTime);
```

Comment:
This is in milliseconds I believe. Also, I think the style is snakeCase (even though not everything here follows it).
```suggestion
          let deltaElapsedMs = msInSec * (elapsed_times[0] - prevElapsedTime);
```